test/integration/default/service.rb

```diff
@@ -1,7 +1,29 @@
 # frozen_string_literal: true
 
-describe service('atlantis') do
-  it { should be_installed }
-  it { should be_enabled }
-  it { should be_running }
+control 'Atlantis Service Common' do
+  describe service('atlantis') do
+    it { should be_installed }
+    it { should be_enabled }
+    it { should be_running }
+  end
+end
+
+control 'Atlantis Service SystemD' do
+  only_if { os.release != '14.04' }
+
+  describe ini('/etc/systemd/system/atlantis.service') do
+    its('Unit.Description') { should cmp 'Atlantis' }
+    its('Unit.After') { should cmp 'network-online.target' }
+
+    its('Service.Type') { should cmp 'simple' }
+    its('Service.User') { should cmp 'atlantis' }
+    its('Service.Group') { should cmp 'atlantis' }
+    its('Service.Environment') { should cmp 'Tom=Roof' }
+    its('Service.ExecStart') { should cmp '/usr/local/bin/atlantis server --config /opt/atlantis/atlantis.yaml' }
+    its('Service.TimeoutStopSec') { should cmp 5 }
```

Comment:
oops, sorry missed that part. Might want to remove that with 0.15 as we don't know how long the stop command will take since it waits for the current terraform command to finish